code/modules/organs/blood.dm

```diff
@@ -125,11 +125,23 @@ var/const/BLOOD_VOLUME_SURVIVE = 40
 
 		//Bleeding out
 		var/blood_max = 0
+		var/blood_loss_divisor = 30	//lower factor = more blood loss
+		//This 30 is the "baseline" of a cut in the "vital" regions (head and torso).
 		for(var/obj/item/organ/external/temp in organs)
 			if(!(temp.status & ORGAN_BLEEDING) || (temp.robotic >= ORGAN_ROBOT))
 				continue
 			for(var/datum/wound/W in temp.wounds)
 				if(W.bleeding())
+					if(W.damage_type == PIERCE) //gunshots and spear stabs bleed more
+						blood_loss_divisor -= 5
+					if(W.damage_type == BRUISE) //bruises bleed less
+						blood_loss_divisor += 5
+					//the farther you get from those vital regions, the less you bleed
+					//depending on how dangerous bleeding turns out to be, it might be better to only apply the reduction to hands and feet
+					if((temp.organ_tag == ARM_LEFT) || (temp.organ_tag == ARM_RIGHT) || (temp.organ_tag == LEG_LEFT) || (temp.organ_tag == LEG_RIGHT))
```

Comment:
It seems really kinda abstract to assume the bleeding reduces. For bruises and such it makes sense, but for the less a lot less so. It would perhaps be more interesting to keep it as is here for bruises but then add the possibility for piercing/cuts to not get reduced, through a `prob` of some sort, because there should still be a chance for other limbs to bleed a lot. It isn't really believably that bleeding just reduces on different limbs.